index.js

```diff
@@ -62,11 +66,11 @@ module.exports = {
 			message: '`with` is disallowed in strict mode because it makes code impossible to predict and optimize.',
 			selector: 'WithStatement',
 		}],
-		'no-tabs': ['error', {
-			allowIndentationTabs: true,
-		}],
+		'no-tabs': 'off',
 		'no-underscore-dangle': ['off'],
 		'no-unused-vars': ['warn'],
+		'object-curly-newline': 'off',
```

Comment:
I prefer the `object-curly-newline: ["error", { "multiline": true }]` [option](https://eslint.org/docs/rules/object-curly-newline#:~:text=Examples%20of%20correct%20code%20for%20this%20rule%20with%20the%20%7B%20%22multiline%22%3A%20true%20%7D%20option%3A)

Reply:
(ie. if one key is on a newline, all should be on newlines)